tests/fpga/long_long_opencl_test.py

```diff
@@ -39,17 +33,13 @@ def run_simple_add(device_type: dace.dtypes.DeviceType):
     '''
 
     # Initialize data (polybench small size)
-    N = sizes["small"]
+    N = 120
     A, B = initialize(N)
     A_ref = np.copy(A)
     B_ref = np.copy(B)
 
-    if device_type in {dace.dtypes.DeviceType.CPU, dace.dtypes.DeviceType.GPU}:
-        # Parse the SDFG and apply auto-opt
-        sdfg = simple_add_kernel.to_sdfg()
-        sdfg = auto_optimize(sdfg, device_type)
-        sdfg(A, B, N=N)
-    elif device_type == dace.dtypes.DeviceType.FPGA:
+
+    if device_type == dace.dtypes.DeviceType.FPGA:
```

Comment:
You can also remove this check on the Device Type, and the use of `device_type` at all. This will be only executed for Intel FPGA anyway